angular/projects/admin/src/app/admin/actions/action-button/action-button.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+export type ActionButtonType = 'delete' | 'save';
+
+
+@Component({
+  selector: 'app-action-button',
+  templateUrl: './action-button.component.html',
+  styleUrls: ['./action-button.component.css']
+})
+export class ActionButtonComponent implements OnInit {
+
+  readonly BUTTON_COLORS = {
+    delete: 'warn',
+    save: 'primary',
+    editTemplate: '',
+    cancel: 'warn'
+  };
+
+  readonly BUTTON_TITLES = {
+    delete: 'Delete',
+    save: 'Save',
+    editTemplate: 'Edit Template',
+    cancel: 'cancel'
```

Comment:
Captitalize the word for consistency

Reply:
Updated in https://github.com/oddbit/tanam/pull/41/commits/c2291eb41877abb5ea8cb12a76f4323cfd899106